cumulus/parachains/common/src/genesis_config_helpers.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Some common helpers for declaring runtime's presets
+// note: copied from: cumulus/polkadot-parachain/src/chain_spec/mod.rs
+
+use crate::{AccountId, Signature};
+use sp_core::{Pair, Public};
+use sp_runtime::traits::{IdentifyAccount, Verify};
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+
+/// The default XCM version to set in genesis config.
+pub const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
```

Comment:
@michalkucharczyk I would suggest removing `SAFE_XCM_VERSION` from here, and move it to the https://github.com/`cumulus/parachains/runtimes/constants` as a shared constant just for **testnets**.

`parachains-common` should not contain any specific constants or values, because we reuse these in the fellows repo. So `parachains-common` should contain just generic functionality.

Relates to: https://github.com/paritytech/polkadot-sdk/pull/3134 or https://github.com/paritytech/polkadot-sdk/issues/3054